modules/oadp-pod-crash-set-resource-request.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: CONCEPT
+[id="oadp-pod-crash-resource-request_context}"]
```

Comment:
@RichardHoch 
One curly bracket is missing. That's why the checks failed. 
[id="oadp-pod-crash-resource-request_{context}"]